ev3dev2/led.py

```diff
@@ -346,3 +385,231 @@ def all_off(self):
 
         for led in self.leds.values():
             led.brightness = 0
+
+    def reset(self):
+        """
+        Put all LEDs back to their default color
+        """
+
+        if not self.leds:
+            return
+
+        for group in self.led_groups:
+            self.set_color(group, LED_DEFAULT_COLOR)
+
+    def animate_stop(self):
```

Comment:
I initially read this name as "animate the LEDs stopping" rather than "stop the LED animation"... can we rename this to something like `stop_animation`?

Reply:
I guess it fits with the names of the other animation methods, so maybe it's better as-is. Up to you.